pkg/controllers/management/secretmigrator/clusters.go

```diff
@@ -957,17 +960,10 @@ func (h *handler) migrateClusterSecrets(cluster *apimgmtv3.Cluster) (*apimgmtv3.
 				}
 			}
 		}
-
-		logrus.Tracef("[secretmigrator] setting cluster condition [%s] and updating cluster [%s]", apimgmtv3.ClusterConditionSecretsMigrated, clusterCopy.Name)
```

Comment:
This update has been removed, as I was able to confirm that in the event of a migration, the two possible scenarios occur:

1. There is no actual cluster update due to a lack of secrets to migrate, except for the corresponding condition being set as `"Unknown"`. In this case, updating to `"Unknown"` to update immediately to `"True"` can be optimized away.
2. The cluster has been updated in one of the `migrateSecret` functions. In this case, the update call will be unneeded, as the cluster will already be the most up to date in etcd.